Makefile.checker

```diff
@@ -75,8 +75,9 @@ checker/%.cmx: checker/%.ml
 
 md5chk:
 	$(SHOW)'MD5SUM cic.mli'
-	$(HIDE)if grep -q `$(MD5SUM) checker/cic.mli` checker/values.ml; \
-	       then true; else echo "Error: outdated checker/values.ml"; false; fi
+	$(HIDE)v=`tr -d "\r" < checker/cic.mli | $(MD5SUM) | sed -n -e 's/ .*/ checker\/cic.mli/' -e '/^/p'`; \
```

Comment:
Whichever character appears first after `s` is used as separator, a good pick avoids the need to escape `/`.
When `/` needs to appear in the expression I usually see `:` as separator, so 
`'s: .*: checker/cic.mli:'`